js_modules/dagit/packages/core/src/pipelines/SidebarModeSection.tsx

```diff
@@ -23,6 +23,7 @@ export const SidebarModeSection: React.FC<{
           <Icon name="resource" color={Colors.Gray700} />
           <div>
             <ContextResourceHeader>{resource.name}</ContextResourceHeader>
+            <ResourceSourceTag>{resource.source || NO_DESCRIPTION}</ResourceSourceTag>
```

Comment:
rather than just showing the enum value straight up here, can we put something human readable?

consider:
```
{resource.source === ResourceSource.FROM_DEFAULT ? (<Tag>default</Tag>) : null}
```

And maybe that links to the repository default resources tab.  It's not clear to me if there's a lot of value in showing the "override" tag.